### PR TITLE
update knox dep.  new version has not yet been published

### PR DESCRIPTION
**Pull request by boneskull**
see #393 and OpenCollective/opencollective#107

### CODE REVIEW

**Comment by boneskull** [comment]
this includes nsp-related code, so you can compare the CI result of #393 to the result of this build.

File: package.json [hunk 1]
```diff
@@ -67,6 +67,7 @@
     "mocha": "2.4.5",
     "mocha-circleci-reporter": "0.0.1",
     "nock": "2.18.2",
+    "nsp": "^2.4.0",
```
Comment (sedubois) [inline package.json]: Can you freeze the version?

Comment (boneskull) [inline package.json]: yes

**Comment by sedubois** [comment]
With this merged, could add https://snyk.io/test/github/opencollective/opencollective-api to README

**Pull request merged**